### ISSUE TITLE
[jsx] add class name for function arguments

### ISSUE THREAD

**Issue by maor-benami** [ajaxorg/ace#4936]
### Describe the feature

 add class name for function arguments to target with color scheme

<img width="656" alt="image" src="https://user-images.githubusercontent.com/85426848/192171528-ae2c7145-7602-4a17-a197-0030b1f2dd6f.png">

### Use Case

i want to enhance my color scheme and give unique color to function arguments

### Proposed Solution

add class like .ace_argument

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### ACE version used

10

**Comment by TheSelfAtu** [comment]
Hi! let me confirm one thing.

function in this issue title means arrow function?
And we should give unique color to arrow function arguments?

Args with Function declaration (not arrow function) has color for now. 
(items, nada is yellow)
![image](https://user-images.githubusercontent.com/43713568/197346750-ba21fe3b-704d-47c8-b9ce-f5c042df6d6f.png)

**Comment by maor-benami** [comment]
In JavaScript mode, the params get "ace_variable ace_parameter" class, but not in JSX mode.

**Comment by TheSelfAtu** [comment]
In above image , How can I tell  (e)  is function args or not ?

 e.g. 
```
flag = true

<Component
  flag={flag}  // flag is not function arg
  onSubmit={e => {}}  // e is function arg
/>

```

**Comment by andrewnester** [comment]
Released in 1.13.2
